src/Platform/Microsoft.Testing.Platform/Hosts/TestHostBuilder.cs

```diff
@@ -236,8 +237,30 @@ public async Task<ITestHost> BuildAsync(
             Logging.AddProvider((_, _) => loggingState.FileLoggerProvider);
         }
 
-        // Register the server mode log forwarder if needed. We follow the console --diagnostic behavior.
```

Comment:
Why is this removed?

Reply:
it's still there but after the code i added